packages/core/src/RenderPipeline/SpriteMaterial.ts

```diff
@@ -1,40 +1,40 @@
 "use strict";
 
-import { Shader } from "@oasis-engine/core";
+import { Shader } from "../shader";
 
 const spriteVertShader = `
 precision highp float;
 
-uniform mat4 matProjection;
-uniform mat4 matView;
+uniform mat4 u_projMat;
```

Comment:
you can use "viewProject" uniform to improve performance

Reply:
> you can use "viewProject" uniform to improve performance

done